src/Microsoft.Extensions.DependencyModel/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "description": "Abstractions for reading `.deps` files.",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Same question here about wanting a pre-release number.

Also, this package exists now in core-setup. So let's make sure the version numbers don't conflict.